XenAdmin/Alerts/Types/MessageAlert.cs

```diff
@@ -240,6 +240,11 @@ public override string Description
                         VMSS vmss = Helpers.XenObjectFromMessage(Message) as VMSS;
                         var policyAlertVMSS = new PolicyAlert(Message.priority, Message.name, Message.timestamp, Message.body, (vmss == null) ? "" : vmss.Name());
                         return policyAlertVMSS.Text;
+
+                    case Message.MessageType.unknown when Message.name == "GFS2_CAPACITY":
+                        if (XenObject != null)
+                            return string.Format(Message.FriendlyBody(Message.name), XenObject.Name());

```

Comment:
Are there any reasonable situations where we could get this error but not a set `XenObject` for which we would want to raise the error anyway but with `Unknown` as a stand-in for the `XenObject`?

Reply:
Throughout XenCenter objects can happen to be null, often when we concincide with a cache update, but in this case we just avoid doing things with them rather than saying something occurred on an unknown object.